src/Components/Bookshelf/index.js

```diff
@@ -2,11 +2,14 @@ import React, { useEffect, useState } from "react"
 import "./bookshelf.css"
 import BookCard from "../BookCard"
 
-function Bookshelf() {
+function Bookshelf({ claimed }) {
     const [bookCollection, setBookCollection] = useState([])
 
     useEffect(() => {
-        fetch("https://book-swap-api.dev.io-academy.uk/api/books")
+        fetch(
+            "https://book-swap-api.dev.io-academy.uk/api/books?claimed=" +
+                claimed
+        )
```

Comment:
There is now a chance that there won't be any books, if all books are claimed/unclaimed, then one of the pages runs the risk of being book-less. 

Worth thinking about displaying a message if the bookCollection state is an empty array